src/components/Card/Card.stories.mdx

```diff
@@ -53,25 +58,39 @@ import { getCategory } from "../../utils/componentCategories";
 | Component Version | DS Version |
 | ----------------- | ---------- |
 | Added             | `0.24.0`   |
+| Latest            | `0.24.0`   |
 
 <Description of={Card} />
 
-Cards are contained units of information that can stand independently from other elements for context. While a Card is a self-contained unit, Cards are often grouped together and displayed in some type of grid layout.
+To quote Material Design: "_A card is identifiable as a single, contained unit. A card can stand alone, without relying on surrounding elements for content._" Additionally, while a `Card` is a self-contained unit, `Card` components are often grouped together and displayed in some type of grid layout.
```

Comment:
Maybe link to `Material Design`? Is this the only place where we are saying that this is influenced by another component library?

Reply:
I rewrote that block of text.